Check git repo on launch prior to process loop.

e-Thug opened the pull request:
## Pull request type

Please check the type of change your PR introduces:

- [ ] Update
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes

## What is the new behavior?

Please describe the behavior or changes that are being added by this PR.

- Really shit implementation of checking the git repo prior to the proc loop begins. If it's old it opens the browser to download the new version, won't auto update because.... lazy.

e-Thug committed:
Update MainWindowViewModel.cs

Checks releases repo for updates prior to pid loop.
Pretty shit implementation, but requests go through as a wii. So...

e-Thug committed:
Update MainWindowViewModel.cs

szaaamerik commented on Forza-Mods-AIO/Forza-Mods-AIO/ViewModels/Windows/MainWindowViewModel.cs:
```diff
@@ -108,10 +113,74 @@ private void InitializeViewModel()
+        {
+            if (Version.Parse(latestGit) > assemblyVersion)
+            {
+                MessageBox.Show($"Update to version {latestGit}", "Update Available", MessageBoxButton.OK, MessageBoxImage.Warning);
```
can you make this a yes/no prompt and if user chooses no, then just don't update?

e-Thug replied:
Updated. Linked the latest commit, don't mind me messing up my copy from vs to git lol
[](url)https://github.com/ForzaMods/Forza-Mods-AIO/pull/41/commits/bf059f827abf61ce03a8c4c6779c5df4c7283637

e-Thug committed:
Update MainWindowViewModel.cs

Updated update message to yes or no.
Yes will go to repo and close the aio, no will do nothing.

e-Thug committed:
Update MainWindowViewModel.cs

szaaamerik commented:
sick! I'll merge
